File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/session/model/SessionFeedback.kt
package pt.isel.leic.ptgest.domain.session.model

import pt.isel.leic.ptgest.domain.common.Source
import java.util.*

data class SessionFeedback(
    val id: Int,
    val source: Source,
    val feedback: String,
    val date: Date
)

File: code/js/src/views/user/TrainerViews/models/sessions/SessionFeedback.ts
class SessionFeedback{
    id: number
    source: string
    feedback: string
    date: Date

    constructor(id: number, source: string, feedback: string, date: Date){
        this.id = id
        this.source = source
        this.feedback = feedback
        this.date = date
    }
}
export default SessionFeedback
